README.md

```diff
@@ -39,12 +39,22 @@ Available targets:
   yaml/lint                           Lints YAML files
 ```
 
-## How to use
+## Prerequisite
```

Comment:
Let's keep the level-2 heading as "How to use", and add a new level-3 heading for the Prerequisites...
```suggestion
## How to use
```